controllers/openstackcontrolplane_controller.go

```diff
@@ -44,6 +44,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// OSPVersion - global var to specify tge OSP version, defaults to OSP17
+var OSPVersion = ospdirectorv1beta1.TemplateVersionWallaby
```

Comment:
I think we should consider defaulting to OSP 16.2 for now to match other settings in the Operator.

Reply:
done